pkg/helm/release/manager.go

```diff
@@ -289,6 +292,17 @@ func generatePatch(existing, expected runtime.Object) ([]byte, error) {
 		return nil, nil
 	}
 
+	// We should add a "remove" operation if the same path does also have
+	// a "add" operation
+	for _, rop := range removeOps {
+		for _, pop := range patchOps {
+			if rop.Path == pop.Path && pop.Operation == "add" {
+				patchOps = append(patchOps, rop)
+				break
+			}
+		}
+	}
+
```

Comment:
Would you mind submitting the jsonpatch fix in a separate PR so we can separate features and bug fixes?

Reply:
Moved to #2777